stixpy/vis/aia_reprojection.py

```diff
@@ -0,0 +1,235 @@
+        end_day : string
+            Provide a string of a the end date with YYYY-DD-MM formatting. e.g '2020-03-10'
+        Returns
+        -------
+        p : tuple
+            position of SOLO in SOLO_HEE coordinate frame
+    """
+    # Specifiying year, month, day
+    start_year = int(start_day[0:4])
+    start_month = int(start_day[5:7])
+    start_day = int(start_day[8:10])
+
+    from pathlib import Path
+    mkp = Path(stixcore.ephemeris.manager.__file__).parent.parent / 'data' / 'test' / 'ephemeris' / 'test_position_20201001_V01.mk'
+    with Position(meta_kernel_path=mkp) as pos:
+        p = pos.get_position(date=datetime(start_year, start_month, start_day), frame='SOLO_HEE')
+    print(p)
+    return(p)
+#__________________________________
+
+
+def convert_SOLO_coords(start_day, end_day, p, aiamap):
+    """
+        Returns the position of SOLO in HeliographicStonyhurst frame and returns the reference frame (AIA as seen by SOLAR ORBITER)
+        Parameters
+        ----------
+        start_day : string
+            Provide a string of the start date with YYYY-DD-MM formatting. e.g '2020-03-09'
+        end_day : string
+            Provide a string of a the end date with YYYY-DD-MM formatting. e.g '2020-03-10'
+        p : tuple
+            Position of SOLO in SOLO_HEE coordinate frame
+        aiamap: SunPy map
+            AIA map
+        Returns
+        -------
+        solo_hgs_ref_coord : SkyCoord
+            The reference frame for the reprojection (AIA as seen by SOLAR ORBITER)
+        solo_hgs : SkyCoord
+            The position of SOLO in HeliographicStonyhurst frame
+    """
+    # Converting return of STIXCORE POISITION to HeliocentricEarthEcliptic frame
+    solo_hee = SkyCoord(*p, frame=frames.HeliocentricEarthEcliptic, representation_type='cartesian', obstime=start_day)
+
+    # Converting HeliocentricEarthEcliptic coords of SOLAR ORBTER position to HeliographicStonyhurst frame
+    solo_hgs = solo_hee.transform_to(frames.HeliographicStonyhurst)
+
+    # Creating reference frame (AIA as seen by SOLAR ORBITER)
+    solo_hgs_ref_coord = SkyCoord(aiamap.reference_coordinate.Tx,aiamap.reference_coordinate.Ty,obstime=aiamap.reference_coordinate.obstime,
+    observer=solo_hgs,frame="helioprojective")
+    return (solo_hgs_ref_coord, solo_hgs)
+#__________________________________
+
+
+def create_wcs_header(solo_hgs_ref_coord, aia_map):
+    """
+        Returns the header, wcs header, and the shape of the reprojected map
+        ----------
+        solo_hgs_ref_coord : SkyCoord
+            The reference frame for the reprojection (AIA as seen by SOLAR ORBITER)
+        solo_hgs : SkyCoord
+            The position of SOLO in HeliographicStonyhurst frame
+        Returns
+        -------
+        solo_wcs : WCS
+            wcs header
+        out_shape : tuple
+            the shape of the reprojected map
+        solo_header : MetaDict
+            header
+    """
+    out_shape = (512, 512)
```

Comment:
maybe add this as keyword argument so it can be configured with a default of `(512, 512)`